src/api/app/controllers/source_controller.rb

```diff
@@ -360,13 +360,13 @@ def package_command
       @spkg = Package.get_by_project_and_name(origin_project_name, origin_package_name)
     end
     # rubocop:enable Metrics/LineLength
-    unless Package_creating_commands.include? @command and not Project.exists_by_name(@target_project_name)
+    unless Package_creating_commands.include? @command && !Project.exists_by_name(@target_project_name)
```

Comment:
Because christian requested them? And I don't want to mix it with the real fix, since I won't backport them for 2.7.

I see that the style change commit is breaking test suite atm, I will check later.